cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -524,6 +524,10 @@ spec:
               value: {{ (quote .Values.kubecostModel.warmSavingsCache) | default (quote true) }}
             - name: ETL_ENABLED
               value: {{ (quote .Values.kubecostModel.etl) | default (quote true) }}
+            {{- if .Values.kubecostModel.etlReadOnlyMode }}
```

Comment:
Ah, it does. I think that's a fair point that this is a misnomer since reconciliation will still ocur. Any naming suggestions?

Reply:
I agree this is a tough one to name. I guess we should've renamed at the cost-model level first since the env var has the same name... I suppose it's better to keep the consistency.